engine/table/src/main/java/io/deephaven/engine/table/impl/select/InstantRangeFilter.java

```diff
@@ -105,5 +105,56 @@ public void filter(Chunk<? extends Values> values, LongChunk<OrderedRowKeys> key
                 longFilter.filter(writableLongChunk, keys, results);
             }
         }
+
+        @Override
+        public int filter(
```

Comment:
could combine the methods, even if we use a method reference it is 1x/chunk not 1x/cell